docs/tasks/20260214_0938_全体ビュー再委託先ノード追加設計.md

```
@@ -0,0 +1,269 @@
+
+---
+
+## 1. Why（目的）
+
+現在、再委託先の情報は**支出先ビュー**（特定の支出先を選択した後）でのみ確認できる。
+
+全体ビューでは「政府 → 府省庁 → 事業 → 支出先」までしか表示されず、支出先がさらにどこに委託しているかが分からない。
+
+全体ビューに再委託先ノードを追加することで、**予算全体のフローを最後の委託先まで一気通貫で把握**できるようになる。
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)